Dockerfile.prod

```diff
@@ -0,0 +1,31 @@
+FROM node:18-alpine AS build
```

Comment:
Please provide the link for this code file from where you have taken inspiration